### PR TITLE
Feature/refresh permission key expired

### PR DESCRIPTION
**Pull request by renejal**

### CODE REVIEW

File: dojo/engagement/views.py [hunk 1]
```diff
@@ -1823,7 +1823,7 @@ def accept_risk_acceptance(request, eid, raid):
     eng = get_object_or_404(Engagement, pk=eid)
     product = eng.product
     product_type = product.get_product_type
-    rp_helper.accept_risk_pending_bullk(eng, risk_acceptance, product, product_type)
+    rp_helper.accept_or_reject_risk_bulk(eng, risk_acceptance, product, product_type)
```
Comment (Gaviria9601) [inline dojo/engagement/views.py]: The parameters action and permission_key are required.

File: dojo/risk_acceptance/risk_pending.py [hunk 2]
```diff
@@ -177,33 +192,50 @@ def risk_acceptante_pending(eng: Engagement,
...
-        message = "Finding Accept successfully from risk acceptance."
-        status = "OK"
+        status_permission.update(
+            {"stasus": "OK",
+             "message": "Finding Accept successfully from risk acceptance.",
+             "number_of_acceptors_required": number_of_acceptors_required})
+    return status_permission
+        
 
+def risk_acceptante_pending(eng: Engagement,
```
Comment (Gaviria9601) [inline dojo/risk_acceptance/risk_pending.py:202]: function is failing with the message
“local variable 'state' cannot be accessed where it is not associated with a value”.

**Review by Gaviria9601 — COMMENTED** [review]
Please, review comments.

File: dojo/risk_acceptance/risk_pending.py (hunk 2 above)
Comment (renejal) [reply dojo/risk_acceptance/risk_pending.py:202]: OK

File: dojo/engagement/views.py (hunk 1 above)
Comment (renejal) [reply dojo/engagement/views.py]: OK

**Review by Gaviria9601 — APPROVED** [review]
👍 Goob job.

**Pull request merged**